internal/network/command_peering.go

```diff
@@ -0,0 +1,69 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+)
+
+type peeringCommand struct {
```

Comment:
I see two approaches in the code base for this.

[command_consumer.go in the stream-share package](https://github.com/confluentinc/cli/blob/main/internal/stream-share/command_consumer.go) is using `command` from [command.go](https://github.com/confluentinc/cli/blob/dc3ab0bcd3b7a2073c77acbe2f432a0856afc804/internal/stream-share/command.go#L9)

[command_cluster.go in the kafka package](https://github.com/confluentinc/cli/blob/dc3ab0bcd3b7a2073c77acbe2f432a0856afc804/internal/kafka/command_cluster.go#L29) defines its own `clusterCommand`.

Which one should we use?

Reply:
i think it's of better practice to use `peeringCommand` which is more specific. Usually we enforce using `<use>Command` when the struct get more complex. Good catch though.

Reply:
Thank you! I will keep `peeringCommand` and follow this convention for the rest of our commands.